### ISSUE TITLE
latexindent package broken - missing dependency YAML::Tiny 

### ISSUE THREAD

**Issue by ChadBailey** [MiKTeX/miktex#1117]
- [x] I have read and acknowledged this HOWTO article:  
  [Submitting a MiKTeX problem report](https://miktex.org/howto/bug-report)

After I filed a detailed bug report on the child project https://github.com/cmhughes/latexindent.pl/issues/374 it appears to me like the owner of this project may not be the maintainer of this package.

In his words:

> I've tried repeatedly to influence change on Miktex, but haven't been successful.

By publishing the name and code of another project you are being a representative for them so you must handle issues with the same care and dignity of that project. If you don't allow them to manage their own package and don't allow them input to your project I would say that is somewhere between rude and completely unethical. I'm not here to point fingers or judge though, just thought it was a point worth making clear. 

I've written up a very detailed bug report, I think it may have all you need. If it doesn't, I may attempt re-breaking my installation, but for now I cannot get the logs the HOWTO requests as the issue has been resolved. I am just posting this in good faith hoping that the package will be fixed or removed if it is indeed broken. Broken packages hurt the package index, source project, and user community so it's better to not have at all than it be broken.

**Comment by cmhughes** [comment]
Please do let me know if I can do anything to help.
Chris (author of latexindent)

**Comment by edocevoli** [comment]
You can fix the issue by installing missing Perl dependencies. That said, run the Perl package manager and install `YAML::Tiny`.

**Comment by ChadBailey** [comment]
This isn't a support request, it's a bug report. I have already solved my issue and provided detailed instructions on how to resolve. Do with this information what you will, I'm just doing my part by alerting you to the unexpected behavior.

I'm really trying to understand the rationale here, but from my (perhaps wrong) understanding it seems like you are essentially agreeing the package is broken and closing the bug report with no work so I must admit I'm confused.

I think the root of the issue may be that I didn't understand having Perl dependent packages was a prerequisite for MiKTeX. I would argue this is a reasonable assumption as Perl and dependent packages are not listed on the [prerequisites page for MiKTeX](https://miktex.org/kb/prerequisites-2-9).

I am on Windows, albeit using [Cygwin](https://www.cygwin.com/). I have already resolved the issue for myself I just wanted to alert you all that the package is broken. When I manually bypassed MiKTeK's copy of latexindex and installed it using the instructions provided by latexindex it worked as expected. This is why I'm calling the package broken, even if installing Perl & `YAML::Tiny` would ultimately resolve the issue. _(I mentioned all of these details in the original bug report)_
